packages/web/src/components/nav/desktop/PlaylistLibrary.module.css

```diff
@@ -16,23 +22,34 @@
   text-overflow: ellipsis;
 }
 
-.iconFolder path {
-  align-self: flex-end;
-  fill: none;
-  stroke-width: 1px;
-  stroke: var(--nav-column-link);
```

Comment:
Double checking - this isn't needed anymore?

Reply:
yupp it all moved to FormActions